src/api/ApiClient.js

```diff
@@ -3,16 +3,44 @@ import base64 from 'base-64';
 
 const apiVersion = 'api/v1';
 
+export type Account = {
+  accountId: number,
+  activeBackend: 'dev' | 'google' | 'password',
+  apiKey: string,
+  authBackends: string[],
+  email: string,
+  directAdmins: string[],
```

Comment:
Yeah, there are two ways to make something 'optional'.
* email?: string - email is optional
* email: ?string - email is mandatory but nullable